Gemfile

```diff
@@ -4,7 +4,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '2.6.6'
 
 # Service/framework dependencies
-gem 'rails', github: "rails/rails", branch: "master" # , '~> 6.0.2', '>= 6.0.2.2'
+# gem 'rails', github: "rails/rails", branch: "master" # , '~> 6.0.2', '>= 6.0.2.2'
+gem 'rails', '~> 6.0.2', '>= 6.0.2.2'
```

Comment:
Can you also see if this works on master again now pls? I don't massively want to downgrade to `6.0`

Reply:
The problem was not with it not working, but with the download being horribly slow and flaky on my Docker for Windows setup. I'll revert.